packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -104,6 +104,23 @@ String getAvdPath() {
   );
 }
 
+/// Locate 'avdmanager'. Prefer to use one from an Android SDK, if we can locate that.
+/// This should be used over accessing androidSdk.avdManagerPath directly because it
+/// will work for those users who have Android Tools installed but
+/// not the full SDK.
+String getAvdManagerPath([AndroidSdk existingSdk]) {
```

Comment:
use a named param instead?

Reply:
This matches other functions in this file. Should we change them to keep them consistent?

```dart
String getAdbPath([AndroidSdk existingSdk]);
String getEmulatorPath([AndroidSdk existingSdk]);
```

Reply:
No, let's keep them consistent.